plugins/accelerated-peft/src/fms_acceleration_peft/autogptq_utils.py

```diff
@@ -35,11 +35,28 @@
 PEFT_ALL_LINEAR = "all-linear"
 
 
-def requires_installation_on_all_linears(peft_config):
+def requires_installation_on_all_linears(peft_config, model_type: str = None):
     tm = peft_config.target_modules
+
+    if tm is None:
+        try:
+            # Third Party
+            # pylint: disable=import-outside-toplevel
+            from peft.utils.constants import (
+                TRANSFORMERS_MODELS_TO_LORA_TARGET_MODULES_MAPPING,
+            )
+
+            tm = TRANSFORMERS_MODELS_TO_LORA_TARGET_MODULES_MAPPING[model_type]
```

Comment:
Hi Fabian, thanks for the quick fix! I was curious - why do we want to set this here when [similar code](https://github.com/huggingface/peft/blob/main/src/peft/tuners/lora/model.py#L467) is called in the PEFT library already?

Reply:
this is because for GPTQ, we do not use the `peft` version of `LoraModel`, we use the `GPTQLoRAModel` https://github.com/foundation-model-stack/fms-acceleration/blob/main/plugins/accelerated-peft/src/fms_acceleration_peft/gptqmodel/utils/peft.py#L46

This. is due to the different handling of the quantized base layer

Reply:
Thanks for the clarification, that's good to know! In that case, this looks good to me but I'd like to get another set of eyes on it before merging.